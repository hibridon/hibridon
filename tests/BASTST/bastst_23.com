inp=bastst_23.inp
out=bastst_23.out
run
save=save_23.inp