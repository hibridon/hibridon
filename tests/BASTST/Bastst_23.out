      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** 3P-2S-AT system parameters:
 NTERM  =   1        NVIB   =   0
 LAMMIN:    1
 LAMMAX:    6
 MPROJ:     0
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= F
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT: -2; JOUT:  0   2
 INDOUT:    3
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-01-24 at 16:06
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: ab01cbf863713480c5e2fb06a97f5320a39d9245
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Jan 24 16:52:25 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500
 ** TOTAL NUMBER OF ANISOTROPIC TERMS= 1

 **  CC 3P/2S ATOM ; NPOT = 0 ** RMU=  16.4700 E=   5000.00
         JTOT=   20   JLPAR= 1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00

 ASYMPTOTIC ATOMIC ENERGY LEVELS
   N   S   J      EINT(CM-1)
   1   3   2        0.000
   2   3   1       43.895
   3   3   0       65.842

 CHANNEL BASIS FUNCTIONS
   N   S   J   J12   L      EINT(CM-1)
   1   3   2    1   20        0.000
   2   3   2    1   22        0.000
   3   3   2    2   18        0.000
   4   3   2    2   20        0.000
   5   3   2    2   22        0.000
   6   3   1    0   20       43.895
   7   3   1    1   20       43.895
   8   3   1    1   22       43.895
   9   3   0    0   20       65.842

 ILAM  LAMBDA  ICOL  IROW   I    IV2    VEE
ILAM=  1 LAMNUM(ILAM) =     21
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS    21
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    5.5 %
      CURRENT DATE:  Jan 24 16:52:25 2023
 ===============================================================================
